src/components/LoansByCategory/HelpmeChoose/HelpmeChooseTrigger.vue

```diff
@@ -33,27 +33,41 @@ export default {
 	computed: {
 		title() {
 			let str = '';
-			if (this.variant === 'amountLeft') {
-				str = 'Almost funded';
-			}
-			if (this.variant === 'personalized') {
-				str = 'Recommended for you';
-			}
-			if (this.variant === 'researchScore') {
-				str = 'Research backed impact';
+			switch (this.variant) {
+				case 'amountLeft':
```

Comment:
Doesn't need to change for this repo, but we'll want to convert these magic strings to a type when we move this code eventually to the `cms-page-server`.